### ISSUE TITLE
Named offset not working as documented

### ISSUE THREAD

**Issue by nirmalkarmani** [iamkun/dayjs#1077]
Based on the advanced format documentation, using z and zzz should give named offsets (abbreviated and non-abbreviated)

https://day.js.org/docs/en/plugin/advanced-format

However, when trying to use z and zzz it doesn't work as specified.

My code (NodeJS with TypeScript)
```typescript
import dayjs from 'dayjs';
import utc from "dayjs/plugin/utc";
import timezone from 'dayjs/plugin/timezone';
import advancedFormat from 'dayjs/plugin/advancedFormat';

dayjs.extend(utc)
dayjs.extend(advancedFormat)
dayjs.extend(timezone)

var out = dayjs().tz('America/Panama');
console.log(out.format('ddd, DD MMM YYYY HH:mm:ss Z z (zzz)'));
```

Expected Output
`Fri, 25 Sep 2020 04:47:06 -05:00 EST (Eastern Standard Time)`

Actual Output
`Fri, 25 Sep 2020 04:47:06 -05:00 z (zzz)`

Node Version : v12.17.0
NPM Version : 6.14.4
dayjs Version: 1.8.36

**Comment by iamkun** [comment]
I'm sorry for the inconvenience, this feature has just been implanted and not released yet.

Will be available in the next release soon.

**Comment by nirmalkarmani** [comment]
Noted, thanks for the prompt response.

Any ETA on the next release? 

Additionally, should I proceed to close this issue? This is the 1st time I have ever reported one and I am not too sure how does one go about after submitting an issue

**Comment by iamkun** [comment]
This feature is released in 1.9.0.

**Comment by nirmalkarmani** [comment]
Hey @iamkun thanks for releasing the update. I have tested v1.9.0 and can confirm it works as expected
